script/Arcueid/Day 7/07_00_ARC07_3G.txt

```diff
@@ -818,25 +821,25 @@ If the lines are blood vessels,
 -- Page 486, Offset 17778.
 -- 「『モノの死に易い線』と『その死』……なんでしょうね、それ。
 // Identical phrase present in 04_00_ARC04_6_2D
-"'The seams of life' and 'death' itself... I wonder what it all means.
+"'Lines where things are easily killed' and 'death' itself... I wonder what it all means.
```

Comment:
Especially this one can do with reverting, see argument above but there's no _ambiguity_ here. It _is_ the "seam of life".